index.html

```diff
@@ -114,10 +114,17 @@ <h2 style="text-align:center;">Currently in use at</h2>
                     <a href="http://www.calpoly.edu"><li class="col"><img src="assets/calpoly.svg" class="greydout" alt="CalPoly" height="60" width="72"></li></a>
                     <a href="http://www.berkeley.edu"><li class="col"><img src="assets/Berkeley.svg" class="greydout" alt="Berkeley" height="60" width="147"></li></a>
                     <a href="http://www.sheffield.ac.uk/"><li class="col"><img src="assets/sheffield.svg" class="greydout" alt="Sheffield" height="60" width="137"></li></a>
+<<<<<<< HEAD
```

Comment:
Sorry, it looks like you still have the merge markers in this file. Can you do:

```
git fetch origin
git checkout origin/master index.html
git commit -m 'restore index.html to master'
```

to reset it back to master?